<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tramos</class>
 <widget class="QWidget" name="Tramos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>N ยบ tramos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_nTramos"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonNtramos">
       <property name="text">
        <string>Introducir nยบ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_punto">
       <property name="text">
        <string>Punto i (x, y): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_x"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_y"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonPunto">
       <property name="text">
        <string>Introducir punto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="ButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
